packages/demos/package.json

```diff
@@ -3,6 +3,13 @@
   "version": "2.8.1",
   "private": true,
   "license": "MIT",
+  "workspaces": {
+    "nohoist": [
+      "@types/webpack-dev-middleware",
+      "webpack",
+      "ts-loader"
+    ]
+  },
```

Comment:
Our augmented types for `webpack` (`packages/demos/types/webpack.d.ts`) are not respected by TypeScript, unless we prevent the two packages that need them (as well as webpack itself) from being hoisted to the root.